frontend/app-development/features/overview/components/Navigation/Navigation.test.tsx

```diff
@@ -50,7 +57,9 @@ describe('Navigation', () => {
     });
 
     topBarMenuItem
-      .filter((item) => item.key !== TopBarMenu.About)
+      .filter(
+        (item) => item.key !== HeaderMenuItemKey.About && item.key !== HeaderMenuItemKey.Deploy,
```

Comment:
As in the previous comment, this is the same filter as line `27` and `41`. 😊

Reply:
See comment above 😄